tests/fixtures/mock_api/api/card/3.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"description": null, "archived": false, "collection_position": null, "table_id": 7, "result_metadata": [{"name": "count", "display_name": "Count", "base_type": "type/BigInteger", "semantic_type": "type/Quantity", "field_ref": ["aggregation", 0], "fingerprint": {"global": {"distinct-count": 1, "nil%": 0.0}, "type": {"type/Number": {"min": 100.0, "q1": 100.0, "q3": 100.0, "max": 100.0, "sd": null, "avg": 100.0}}}}], "can_write": true, "database_id": 2, "enable_embedding": false, "collection_id": 3, "query_type": "query", "name": "Total customers", "dashboard_count": 1, "creator_id": 2, "updated_at": "2021-07-21T08:01:41.996Z", "made_public_by_id": null, "embedding_params": null, "cache_ttl": null, "dataset_query": {"type": "query", "database": 2, "query": {"source-table": 7, "aggregation": [["count"]]}}, "id": 3, "display": "scalar", "last-edit-info": {"id": 1, "email": "user@example.com", "first_name": "Alexander", "last_name": "Butler", "timestamp": "2021-07-21T08:01:37.449936Z"}, "visualization_settings": {"graph.series_labels": ["number"], "graph.metrics": ["count"], "graph.dimensions": []}, "collection": {"authority_level": null, "description": "Automatically generated cards.", "archived": false, "slug": "a_look_at_your_customers_table", "color": "#f9d45c", "name": "A look at your customers table", "personal_owner_id": null, "id": 3, "location": "/2/", "namespace": null}, "created_at": "2021-07-21T08:01:37.434243Z", "public_uuid": null}
```

Comment:
This was changed to exercise the modified code by removing the "creator" key, causing it to look up the creator via id, which does not exist.

Reply:
@z3z1ma You wrote these tests, can you have a look to make sure you're cool with this?

Reply:
@kochalex I created the test mocks using a real scraped metabase instance so I cant tell at a glance but can you just verify the json is modified in such a way that it reflects what metabase returns in the test case we are recreating?

To just delete the owner key but that not be a reflection of reality might impact future development in an unexpected way causing tests that fail when they are valid against metabase API. 

Just checking 🪂

Reply:
This does cause the same error that occurs against a real metabase instance. I'll add an additional card mock API reponse though and try to export it directly from a test metabase deployment.

Here's what was happening in the error cases when running dbt-metabase against on our metabase deployment:
* Metabase's API returns cards that look like this - with only a creator_id but not the creator details.
  *  This is the case for at least some cards where the owner has been deleted (it may be for others as well)
* When dbt-metabase get a card response where there is a creator_id but not the creator details in the response, it makes a call to metabase's user API
  * Metabase's user API filters out deleted users and you are not able to retrieve information on them.
  * dbt-metabase crashes as this point